pkg/controller/storagecluster/cephcluster.go

```diff
@@ -475,3 +484,29 @@ func newCephDaemonResources(custom map[string]corev1.ResourceRequirements) map[s
 
 	return resources
 }
+
+func (r *ReconcileStorageCluster) throttleStorageDevices(storageClassName string) (bool, error) {
+	storageClass := &storagev1.StorageClass{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Namespace: "", Name: storageClassName}, storageClass)
+	if err != nil {
+		return false, fmt.Errorf("failed to retrieve StorageClass %q. %+v", storageClassName, err)
+	}
+	bool flag := false
```

Comment:
`flag` is a terrible name. :) What does this flag represent? Also you probably don't need `bool` at the front.